quic/s2n-quic-qns/Cargo.toml

```
@@ -48,5 +51,5 @@ mimalloc = { version = "0.1", default-features = false }
 normal = [ "openssl-sys" ]
 
 # Use unstable s2n-quic features
-[target.'cfg(s2n_quic_unstable)'.dependencies]
-s2n-quic = { path = "../s2n-quic", features = ["unstable_client_hello", "unstable_resumption"] }
+[target.'cfg(any(s2n_quic_unstable, s2n_internal_dev))'.dependencies]
+s2n-quic = { path = "../s2n-quic", features = ["unstable_client_hello", "unstable_resumption", "unstable-provider-packet-interceptor"] }
```

I think we need to split the features to fix the windows build

```suggestion
[target.'cfg(all(any(s2n_quic_unstable, s2n_internal_dev), unix))'.dependencies]
s2n-quic = { path = "../s2n-quic", features = ["unstable_client_hello", "unstable_resumption"] }

[target.'cfg(any(s2n_quic_unstable, s2n_internal_dev))'.dependencies]
s2n-quic = { path = "../s2n-quic", features = ["unstable-provider-packet-interceptor"] }
```